GTG/plugins/send_email/sendEmail.py

```diff
@@ -32,22 +33,38 @@ class pluginSendEmail:
     """
     The plugin.
     """
+    def __init__(self):
+        """
+        Initialize variables.
+        """
+        self.browser_plugin_api = None
+        self.taskview_plugin_api = None
```

Comment:
You don't need to store plugin_apis because engine will provide it to you:
https://github.com/getting-things-gnome/gtg/blob/master/GTG/core/plugins/engine.py#L237-L248

BTW: if multiple tasks can be opened, you work with assumption there will be just a single task.

Remove those variables.